### ISSUE TITLE
css :has selector not validated correctly when using complex selectors

### ISSUE THREAD

**Issue by Kokujou** [runem/lit-analyzer#374]
the following code causes the lit-plugin to show lots of errors, because it doesn't detect it as valid syntax:

```css
  *:has(> input) { 
...
}
```

of course i'm using latest vscode and lit-plugin version, and if this matters, my lit version is 3.2.0

**Comment by ACregan** [comment]
I too am having this issue. This is a perfectly valid selector:

```css
    .pop-input-label:has(+ .pop-input-input:disabled ) {
      opacity: 0.2;
    }
```

but lit-plugin gives me squigglies with this error:
`) expected`
